### DISCUSSION TITLE
Optimized display of a date range

### DISCUSSION THREAD

**Discussion by lothargrieb** [js-fns/date-fns#2450] [Q&A]
So far I have declared the date range using two "PP" patterns
with the following result (examples in german format):

`16. Apr. 2021 – 22. Apr. 2021`

But now I want to keep the display as short as possible
and also make it more readable:

`16. – 22. Apr. 2021`

or also within a month change:

`22. Apr – 05. Mai. 2021`

At first sight it looks simple, 
but taking into consideration the internationalization is not so easy after all.

Who has an idea?

**Comment by fturmel** [comment]
Hi @lothargrieb,

I answered a similar question earlier today (#2825) and recommended looking at `Intl` to achieve this. It might be helpful to you as well.
